scripts/canbus_query.py

```diff
@@ -37,7 +37,14 @@ def query_unassigned(canbus_iface):
         if uuid in found_ids:
             continue
         found_ids[uuid] = 1
-        sys.stdout.write("Found canbus_uuid=%012x\n" % (uuid,))
+        app = "Klipper"
+        if msg.dlc > 7:
+            if msg.data[7] == 1:
+                app = "CanBoot"
+            elif msg.data[7] != 0:
+                app = "Unknown"
+        sys.stdout.write("Found canbus_uuid=%012x, Application: %s\n"
+                         % (uuid, app))
```

Comment:
Minor, but it may be nice to tie the application id to the SET_NODEID_CMD identifier (thus perhaps fewer "magic values").  Something like:
```python
    AppNames = {CMD_SET_KLIPPER_NODEID: "Klipper", CMD_SET_CANBOOT_NODEID: "CanBoot"}
    ...
    app_id = CMD_SET_KLIPPER_NODEID
    if msg.dlc > 7:
        app_id = msg.data[7]
    app_name = AppNames.get(app_id, "Unknown")
    ...
```